components/breadcrumbs/breadcrumb.js

```diff
@@ -0,0 +1,89 @@
+import '../icons/icon.js';
+import '../link/link.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
```

Comment:
@margaree has been going through and adding JSDoc comments to the classes and properties in core. We're going to use these to build up a documentation site for everything automatically. Would you be able to add this in based on some examples from the other components in core?

Reply:
Sure. Sounds great

Reply:
Added comments based on what I saw on other components. Please let me know if something should be added/removed